config/database.yml

```diff
@@ -1,24 +1,24 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: localhost
-  username: postgres
-  password: mysecretpassword
   pool: 5
   timeout: 5000
+  url: <%= ENV['DATABASE_URL'] %>
 
 development:
   <<: *default
   database: book_swap_development
+  host: localhost
```

Comment:
I think this can also go into the default block, and can be set to 127.0.0.1 (which is the IP address corresponding to localhost). Or is it even necessary? Not really sure.